src/TestFramework/TestFramework/Resources/FrameworkMessages.resx

```diff
@@ -371,6 +371,6 @@ Actual: {2}</value>
     <value>CollectionAssert.ReferenceEquals should not be used for Assertions. Please use CollectionAssert methods or Assert.AreSame &amp; overloads instead.</value>
   </data>
   <data name="ReturnedSubsetValueMessage" xml:space="preserve">
-    <value>Element(s) &lt;{0}&gt; is/are not present in the collection</value>
+    <value>Element(s) &lt;{0}&gt; is/are not present in the collection.</value>
```

Comment:
I'm not sure but I wonder if the enclosing message isn't already adding the final "."

Reply:
This message replaces the `{1}` place holder in:

https://github.com/microsoft/testfx/blob/9086d5ea5fce06fdd36dc44e5c84d282783d79f8/src/TestFramework/TestFramework/Resources/FrameworkMessages.resx#L203-L205